jest.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   maxConcurrency: 1,
   setupFilesAfterEnv: ["jest-extended/all"],
   testTimeout: 60 * 60 * 1000,
+  collectCoverage: true,
```

Comment:
@barnapa This will also enable coverage collection on the CI but for that to be useful it also has to be collected and uploaded from each job and then massaged together (from the different jobs) so that the coverage badge gets updated on codecov.io.
Until all of that happens I suggest to remove this flag for now. You can still use it locally as you see fit.

Reply:
@petermetz Deleted, I think it got omitted while refactoring before PR :)